res/values-pt-rBR/strings.xml

```diff
@@ -69,7 +69,7 @@
     <string name="Delay_Disconnect">"Atrasar a Desconexão"</string>
 
     <string name="Max_Players">"Máx. de Jogadores"</string>
-    <string name="Min_Players">"Mín. de Jogadores"</string>
+    <string name="Min_Players">"Mín. de Jogadores. "</string>
```

Comment:
You still keep changing the wrong language and keep inserting wrong periods. Don't forget you are editing the file `values-pt-rBR`, that is **Brazilian Portuguese**, and for **Vietnamese**, you must edit the `values-vi`!